plugins/module_utils/constants.py

```diff
@@ -211,3 +211,5 @@
 
 ORIGINATE_DEFAULT_ROUTE = {"only": "only", "in_addition": "inAddition", "": ""}
 L3OUT_ROUTING_PROTOCOLS = {"bgp": ["bgp"], "ospf": ["ospf"], "bgpOspf": ["bgp", "ospf"], None: [None], "": None, "bgpospf": "bgpOspf", "ospfbgp": "bgpOspf"}
+
+ENABLED_DISABLED_BOOLEAN_MAP = {"enabled": True, "disabled": False, True: "enabled", False: "disabled"}
```

Comment:
we already have map for this: ENABLED_OR_DISABLED_TO_BOOL_STRING_MAP should we reuse or combine the two maps?